init.lua

```diff
@@ -716,15 +710,29 @@ require('lazy').setup({
         automatic_installation = false,
         handlers = {
           function(server_name)
-            local server = servers[server_name] or {}
-            -- This handles overriding only values explicitly passed
-            -- by the server configuration above. Useful when disabling
-            -- certain features of an LSP (for example, turning off formatting for ts_ls)
-            server.capabilities = vim.tbl_deep_extend('force', {}, capabilities, server.capabilities or {})
-            require('lspconfig')[server_name].setup(server)
+            local config = servers[server_name] or {}
+            vim.lsp.config(server_name, config)
+            vim.lsp.enable(server_name)
           end,
         },
       }
+      -- NOTE: Some servers still require the nvim-lspconfig setup until they are updated
+      -- Add this template inside the handler function after initializing config if you encounter issues with any lsp
+      --
+      -- if server == 'example_server' or server == 'example_server2' then
+      --   -- This handles overriding only values explicitly passed
+      --   -- by the server configuration above. Useful when disabling
+      --   -- certain features of an LSP (for example, turning off formatting for ts_ls)
+      --   local capabilities = require('blink.cmp').get_lsp_capabilities()
+      --   config.capabilities = vim.tbl_deep_extend('force', {}, capabilities, config.capabilities or {})
+      --   require('mason-lspconfig')[server].setup(config)
+      --   return
+      -- end
```

Comment:
Thanks for this contribution, was just trying this out, should it be `server_name` here?
```suggestion
      -- if server_name == 'example_server' or server_name == 'example_server2' then
      --   -- This handles overriding only values explicitly passed
      --   -- by the server configuration above. Useful when disabling
      --   -- certain features of an LSP (for example, turning off formatting for ts_ls)
      --   local capabilities = require('blink.cmp').get_lsp_capabilities()
      --   config.capabilities = vim.tbl_deep_extend('force', {}, capabilities, config.capabilities or {})
      --   require('mason-lspconfig')[server_name].setup(config)
      --   return
      -- end
```